tasks/shared_library_posting_to_queue/shared_lib_posting_to_status_queue.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+import json
+import boto3
+from typing import Dict,Any, Optional
+import time
+
+class RequestMethod(Enum):
+    POST = 'post'
+    PUT = 'put'
+
+class ORCA_STATUS(Enum):
```

Comment:
I'd say switch this to OrcaStatus or RecoveryStatus so our naming scheme is consistent.

Reply:
changed to OrcaStatus